woeip/apps/database/models.py

```diff
@@ -0,0 +1,118 @@
+from django.contrib.gis.db.models import LineStringField, PointField
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+
+
+class Organization(models.Model):
+    """An organization, e.g., WOEIP, OpenOakland"""
+    name = models.CharField(max_length=256)
+    website = models.CharField(max_length=256)
+    email = models.CharField(max_length=256)
+    contact = models.ForeignKey('Participant',
+                                related_name='contact', related_query_name='contact',
+                                on_delete=models.SET_NULL, blank=True, null=True)
+
+    def __str__(self):
+        return f"{self.name} <{self.email}>"
+
+
+class RoleEnum(models.Model):
```

Comment:
What is the purpose of this model?

Reply:
``RoleEnum`` will probably get removed once I figure out how to implement permissions using django's built in functionality.

Reply:
There is a concept called "You Ain't Gonna Need It," abbreviated as YAGNI. The odds are some of the code we think we will need later probably won't be used. So, if you know this will be removed later, don't bother adding it now.

Reply:
"You Ain't Gonna Need it" resonates with me more than "Tables are cheap"